start-build-env.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
```

Comment:
It seems like you've copied this file from trunk. Could you please explain why the changes to this file are needed?

Reply:
@GauthamBanasandra  the line 1: see HADOOP-13963. /bin/bash is hard coded in some of the scripts. Contributed by Ajay Yadav.  and others changes can be see via git log at chunk branch. and besides this, I give up Dockerfile_aarch64, for centos 7 support arm64 platform. so I removal relational codes.

Reply:
`start-build-env.sh` creates the build environment based on `Dockerfile`. Since you haven't modified `Dockerfile` in this PR, you shouldn't change this file as well.

Here's my suggestion to you - let's keep the old way as it is, which is, building using Virtualbox that uses `Dockerfile`. Those who want to use Docker for building can use `Dockerfile_centos_7` directly and not through `start-build-env.sh` -
```shell
$ docker build -t hadoop-centos-7 -f .\dev-support\docker\Dockerfile_centos_7 .\dev-support\docker\
```